### ISSUE TITLE
'time' type WITHOUT date in posgtresql 9.0 resulting complete date/type when it's fetched

### ISSUE THREAD

**Issue by jeremyevans** [jeremyevans/sequel#343]
<b>What steps will reproduce the problem?</b>
1.Create table 'foo' with colume name 'time_test' type 'time' only.
1. insert time data using raw sql (from psql,etc.) no sequel/ruby involved:
   INSERT INTO foo (time_test) values ('16:02:23');
   insert as many data as you'd like if you want
2. after connecting stuff, do soemthing like this
   DB[:foo].each do|row|
   puts row[:time_test]  
   end

<b>What is the expected output? What do you see instead?</b>
EXPECTED:
16:02:23
RESULT:
Sat Dec 25 16:02:23 +0700 2010

The date is the current local date and +0700 is my local time zone.

<b>What version of the product are you using? On what operating system?</b>
Postgresql 9.0.1 ,
pg 0.10.0, 
sequel 3.18.0, 
ruby enterprise edition 1.8.7  
on Linux zenwalk 6.4

<b>Please provide any additional information below.</b>
In short, I expect time only without date but sequel give me full timestamp. The format is similiar like you run 'date' command in Linux.

Google Code Info:
Issue #: 319
Author: satul...@gmx.com
Created On: 2010-12-25T09:29:43.000Z
Closed On: 2010-12-25T17:04:50.000Z
